packages/core/src/awsService/appBuilder/explorer/samProject.ts

```diff
@@ -81,10 +84,12 @@ function getResourceEntity(template: any): ResourceTreeEntity[] {
             Handler: resource.Properties?.Handler ?? template?.Globals?.Function?.Handler,
             Events: resource.Properties?.Events ? getEvents(resource.Properties.Events) : undefined,
             CodeUri: resource.Properties?.CodeUri ?? template?.Globals?.Function?.CodeUri,
+            Environment: resource.Properties?.Environment
+                ? resource.Properties?.Environment
+                : template?.Globals?.Function?.Environment,
```

Comment:
Can we provide this if-else condition like we did for other fields, 
`Environment: resource.Properties?.Environment ?? template?.Globals?.Function?.Environment`